test/docTest.js.html

```diff
@@ -230,103 +234,113 @@
-            var element = $("#add-remove-class-test").addClass("added-class-for-test");
+          it('should add class to the given element', function(){
+            var element = $('#add-remove-class-test').addClass('added-class-for-test');
             assert.equal( element.size, 1);
-            assert.equal( element.hasClass("added-class-for-test"), true);
+            assert.equal( element.hasClass('added-class-for-test'), true);
+          });
+
+          it('should remove class to the given element', function(){
+            var element = $('#add-remove-class-test').removeClass('to-remove-class-for-test');
+            assert.equal( element.hasClass('to-remove-class-for-test'), false);
           });
 
-          it("should remove class to the given element", function(){
-            var element = $("#add-remove-class-test").removeClass("to-remove-class-for-test");
-            assert.equal( element.hasClass("to-remove-class-for-test"), false);
+          it('should toggle class to the given element', function(){
+            var element = $('#toggle-class-test').toggleClass('toggle-class-for-test');
+            assert.equal( element.hasClass('toggle-class-for-test'), true);
+            element = $('#toggle-class-test').toggleClass('toggle-class-for-test');
+            assert.equal( element.hasClass('toggle-class-for-test'), false);
           });
 
-          it("should toggle class to the given element", function(){
-            var element = $("#toggle-class-test").toggleClass("toggle-class-for-test");
-            assert.equal( element.hasClass("toggle-class-for-test"), true);
-            element = $("#toggle-class-test").toggleClass("toggle-class-for-test");
-            assert.equal( element.hasClass("toggle-class-for-test"), false);
+          it('should return html of given element', function(){
+            assert.equal($('#get-html-test').html(), 'My Html');
           });
 
-          it("should return html of given element", function(){
-            assert.equal($("#get-html-test").html(), 'My Html');
+          it('should set html of given element', function(){
+            $('#get-html-test').html('My Html2');
+            assert.equal($('#get-html-test').html(), 'My Html2');
           });
 
-          it("should set html of given element", function(){
-            $("#get-html-test").html('My Html2');
-            assert.equal($("#get-html-test").html(), 'My Html2');
+          it('should prepend a new element and return the given element', function() {
+            var element = document.createElement('div');
+            element.id = 'prepend-element';
+            var prepend = $('#prepend-div').prepend(element),
+              prependElement = prepend.find('#prepend-element');
+            assert.equal(prependElement.size, 1);
+            assert.equal(prependElement.attr('id'), 'prepend-element');
+            assert.equal(prependElement.next().hasClass('this-one'), true);
```

Comment:
It would be good to test for both cases of the prepend (string and element), not only the `element` one.
Also, when a string is passed to the method, you should test for cases where you insert real html `<p>example</p> or just text `example`.

@mottam